packages/next/build/webpack-config.ts

```diff
@@ -368,9 +368,11 @@ export default async function getBaseWebpackConfig(
       // Which makes bundles slightly smaller, but also skips parsing a module that we know will result in this alias
       'next/head': 'next/dist/next-server/lib/head.js',
       'next/router': 'next/dist/client/router.js',
-      'next/experimental-script': config.experimental.scriptLoader
-        ? 'next/dist/client/experimental-script.js'
-        : '',
+      ...(config.experimental.scriptLoader
```

Comment:
This change doesn't seem to have the intended effect, it should fail to resolve `next/experimental-script` when the flag is not enabled hence why it had `''`. Maybe we can just remove this block given it's already named `experimental-script`.

Reply:
@timneutkens yeah, when the flag is not defined it won't add the alias so `next/experimental-script` would fail to resolve. The tests failed because i haven't for some reason put the actual path in the alias 😃 They are passing now, thanks a lot for the input 👍

Reply:
It would resolve fine which is what the existing logic tried to avoid. `packages/next/experimental-script.js` exists.